github/ci/prow/files/jobs/kubevirt/user-guide/user-guide-periodics.yaml

```diff
@@ -0,0 +1,23 @@
+periodics:
+  - interval: 24h
```

Comment:
What is the intent of this job? Is it general validation? If so, maybe rather use a postsubmit that runs everytime a push against `source` occurs.